index.html

```diff
@@ -767,18 +850,155 @@ <h2><dfn>CaptureController</dfn></h2>
             [Exposed=Window, SecureContext]
             interface CaptureController {
               constructor();
-              // TODO: Add setFocusBehavior() in a separate PR.
+              undefined setFocusBehavior(CaptureStartFocusBehavior focusBehavior);
             };
           </pre>
           <dl data-link-for="CaptureController" data-dfn-for="CaptureController" class="methods">
             <dt>
               <dfn>constructor</dfn>
             </dt>
+            <dd>
+              Create a new object with the following internal slots:
+              <table class="simple">
+                <thead>
+                  <tr>
+                    <th>Internal Slot</th>
+                    <th>Initial value</th>
+                    <th>Description (<em>non-normative</em>)</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td><dfn>[[\IsBound]]</dfn></td>
+                    <td><code>false</code></td>
+                    <td>
+                      Tracks whether an application has attempted to associate [=this=]
+                      with a [=capture-session=], by providing it as a value to
+                      {{CaptureController/setFocusBehavior()}}.
+                    </td>
+                  </tr>
+                  <tr>
+                    <td><dfn>[[\Source]]</dfn></td>
+                    <td><code>null</code></td>
+                    <td>
+                      Tracks with which source the [=capture-session=] is associated.
+                      It is distinct from {{CaptureController/[[IsBound]]}}, in that
+                      {{CaptureController/[[IsBound]]}} is set even if
+                      {{MediaDevices/getDisplayMedia()}} rejects, whereas {{CaptureController/[[Source]]}}
+                      will only be set if {{MediaDevices/getDisplayMedia()}} succeeds.
+                    </td>
+                  </tr>
+                  <tr>
+                    <td><dfn>[[\DisplaySurfaceType]]</dfn></td>
+                    <td><code>null</code></td>
+                    <td>
+                      Once capture starts, this will be set to the type of the captured [=display surface=].
+                    </td>
+                  </tr>
+                  <tr>
+                    <td><dfn>[[\InitialFocusLost]]</dfn></td>
+                    <td><code>false</code></td>
+                    <td>
+                      Ensures that calls to {{CaptureController/setFocusBehavior()}} fail
+                      without throwing if the user manually focuses another tab or native application.
+                    </td>
+                  </tr>
+                  <tr>
+                    <td><dfn>[[\FocusWindowClosed]]</dfn></td>
```

Comment:
FocusWindow sounds like it could be a thing. How about:
```suggestion
                    <td><dfn>[[\FocusOpportunityExpired]]</dfn></td>
```